File: apina-core/src/main/java/fi/evident/apina/spring/EndpointParameterNameNotDefinedException.java
package fi.evident.apina.spring;

import fi.evident.apina.java.model.JavaMethod;

public final class EndpointParameterNameNotDefinedException extends RuntimeException {

    public EndpointParameterNameNotDefinedException(JavaMethod method) {
        super("Could not resolve endpoint parameter name of method '" + method.getName() + "' from class " + method.getOwningClass().getName() + ". Add '-parameters' argument for javac.");
    }
}

File: apina-core/src/main/kotlin/fi/evident/apina/spring/EndpointParameterNameNotDefinedException.kt
package fi.evident.apina.spring

import fi.evident.apina.java.model.JavaMethod

class EndpointParameterNameNotDefinedException(method: JavaMethod) : RuntimeException("Could not resolve endpoint parameter name of method '" + method.name + "' from class " + method.owningClass.name + ". Add '-parameters' argument for javac.")
